presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -309,7 +309,7 @@ private void processPage(Page page)
             }
         }
 
-        public boolean isFull()
+        public boolean checkFullAndUpdateMemory()
```

Comment:
I tried splitting this into two functions (`updateMemory` and `isFull`) myself. After doing so, I realize that `updateMemory` does not need to be public. It only need to be called by `processPage`. As a result, splitting doesn't actually change the API as much as I anticipated.

This way, I find it unnecessary to change the name of this method. I will skip this commit. Instead, I'll just split this method up into two, and keep `isFull` as one of the public methods of this class.